bin/src/lib.rs

```diff
@@ -15,11 +22,17 @@ static LOCK_NAME: &str = "iron-wallet";
 static LOCK_NAME: &str = "iron-wallet-dev";
 
 #[tokio::main]
-pub async fn run() -> color_eyre::Result<()> {
+pub async fn run() -> Result<()> {
     ethui_tracing::init()?;
     fix_path_env::fix()?;
 
     let args = ethui_args::parse();
+
+    #[cfg(feature = "forge-traces")]
```

Comment:
all these cfg usages in this file are a bit of an anti-pattern.

instead, you should:

- Add a `App` subcommand to the enum, and make that the default one. meaning that `./ethui` and `./ethui app` are the same thing
- the code you're skipping on `run` just becomes what you run when in the `App` subcommand
- the `Forge` subcommand just gets delegated to the forge traces crate (for now at least). so you can move a lot of this code inside it